<launch>
 <arg name="veh"/>
  <group ns="$(arg veh)">
    <node pkg="fused_localization" type="encoder_localization.py" name="encoder_localization_node" output="screen"/>
    <node pkg="fused_localization" type="at_localization.py" name="at_localization_node" output="screen"/>
    <node pkg="fused_localization" type="fused_localization.py" name="fused_localization_node" output="screen"/>
  </group>

</launch>
